src/OrientationProvider.js

```diff
@@ -31,7 +53,7 @@ export const OrientationProvider = ({ children }) => {
   }, []);
 
   return (
-    <OrientationContext.Provider value={{ dimensions }}>
+    <OrientationContext.Provider value={{ orientation, dimensions }}>
       {children}
     </OrientationContext.Provider>
```

Comment:
OrientationContext.Provider is passing its logic and orientation and dimensions props to all {children} component across the app , all children inside  <MainAppWithApolloProvider />